contrib/for-tests/netexec/netexec.go

```diff
@@ -191,11 +193,53 @@ func dialUDP(request string, remoteAddress *net.UDPAddr) (string, error) {
 
 func shellHandler(w http.ResponseWriter, r *http.Request) {
 	log.Println(r.FormValue("shellCommand"))
-	output, err := exec.Command(shellPath, "-c", r.FormValue("shellCommand")).CombinedOutput()
-	assertNoError(err)
```

Comment:
`assertNoError` calls `log.Fatal` if err is non-nil, which will terminate the executable. Do think that's appropriate, or is it ok to keep things running if there is an error?